api/config.js

```diff
@@ -89,6 +92,9 @@ config.mail.sinkhole = '.sink.sparkpostmail.com';
 config.mail.whitelistedDomains = ['@hackillinois.org'];
 config.mail.whitelistedLists = ['test'];
 
+config.storage.maxLength = '2mb';
```

Comment:
Is this actually used? [Seems like you set this as a constant later on](https://github.com/HackIllinois/api-2017/pull/21/files#diff-73fed693c78424fa355824bae1260449R22).